src/components/form/cancel-button/__spec__.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { elementsTagTest, rootTagTest } from '../../../utils/helpers/tags/tags-specs';
+import { shallow } from 'enzyme';
+import CancelButton from './cancel-button';
+import Button from './../../button';
+
+describe('CancelButton', () => {
+ let wrapper, clickSpy;
+
+ beforeEach(() => {
+   wrapper = shallow(
+     <CancelButton
+       cancelText='This is a cancel button'
+     />
+   );
+ });
+
+  describe('the standard class', () => {
+    it('renders it', () => {
+      let button = wrapper.find('.carbon-form-cancel__button');
+      expect(wrapper.hasClass('carbon-form-cancel')).toBeTruthy();
+    });
+  });
+
+  describe('the save text', () => {
```

Comment:
These describes are a bit cluttered, maybe try:

```
it('renders standard class')

it('renders custom save text if provided')

it('renders default save text if not provided')

it('renders custom class on button')
```

i.e. remove a lot of the unnecessary nested describe stuff that makes the test output harder to read